tools/network/telemetryURIUpdateService.go

```diff
@@ -60,7 +60,7 @@ func (t *telemetryURIUpdater) Start() {
 		updateTelemetryURI := func() {
 			endpointURL := t.lookupTelemetryURL()
 
-			if endpointURL != nil && endpointURL.String() != t.log.GetTelemetryURI() {
+			if endpointURL != nil && endpointURL.String() != t.log.GetTelemetryURI() && false == t.cfg.DisableNetwork {
```

Comment:
Was there a specific problem with telemetry? There might be cases where it would be useful for a single-node deployment and I don't see a technical reason why it would have to be disabled.

Reply:
So, it's true that for a single-node network this isn't an issue. However, the flag is "DisableNetwork", and as such, no outgoing network activity is expected.. including no telemetry.
( note that I didn't want to disable the telemetry since we currently use it for logging to file ).

Reply:
Maybe call the property "DisableNetworking" in that case, I read it as "DisableAlgorandNetwork"